### ISSUE TITLE
Qt5 GUI wrong visual style

### ISSUE THREAD

**Issue by SteffenL** [Microsoft/vcpkg#856]
Hello,

I have rewritten the issue description to reflect my latest discoveries. My original issue was that the visual style in my application looked rather old (Windows 9x) when using Qt5 out of the box.

I do not know why the styles have been excluded from the build, but it seems sensible to me to include them.

## Environment

OS: Windows 10
qt5:x86-windows package version: 5.7.1-6
Compiler/IDE: Visual Studio 2017
CMake version outside of Visual Studio: 3.8.0-rc3

## Steps to reproduce

First some setup:
```
git clone https://github.com/SteffenL/vcpkg-issue-856.git
cd vcpkg-issue-856

git submodule update --init
pushd external\vcpkg
powershell -ExecutionPolicy Bypass -File .\scripts\bootstrap.ps1
.\vcpkg.exe install qt5
popd
```

Build and run:
```
mkdir build && cd build
cmake -G "Visual Studio 15 2017" -DCMAKE_TOOLCHAIN_FILE=external/vcpkg/scripts/buildsystems/vcpkg.cmake ..
cmake --build . --config Release
app\Release\app.exe
```

Result:

![2017-03-29 2](https://cloud.githubusercontent.com/assets/1543854/24473959/5923c1a8-14cb-11e7-934f-1964b5a96638.png)

## Fix

Since I would prefer my application to be rendered properly, I applied these changes with success:

```
diff --git a/ports/qt5/portfile.cmake b/ports/qt5/portfile.cmake
index 56ab9d4..2319c26 100644
--- a/ports/qt5/portfile.cmake
+++ b/ports/qt5/portfile.cmake
@@ -67,9 +67,6 @@ vcpkg_execute_required_process(
         -no-iconv
         -system-sqlite
         -opengl desktop # other options are "-no-opengl" and "-opengl angle"
-        -no-style-windowsxp
-        -no-style-windowsvista
-        -no-style-fusion
         -mp
         -nomake examples -nomake tests -no-compile-examples
         -skip webengine -skip declarative
```

Result after rebuilding everything:

![2017-03-29 1](https://cloud.githubusercontent.com/assets/1543854/24473969/632df11e-14cb-11e7-9e16-f37c5b11a0db.png)

**Comment by SteffenL** [comment]
## How to reproduce the issue

First some setup:
```
git clone https://github.com/SteffenL/vcpkg-issue-856.git
cd vcpkg-issue-856

git submodule update --init
pushd external\vcpkg
powershell -ExecutionPolicy Bypass -File .\scripts\bootstrap.ps1
.\vcpkg.exe install qt5
popd
```

Build:
```
mkdir build && cd build
cmake -G "Visual Studio 15 2017" -DCMAKE_TOOLCHAIN_FILE=external/vcpkg/scripts/buildsystems/vcpkg.cmake ..
cmake --build . --config Release
app\Release\app.exe
```

**Comment by ras0219-msft** [comment]
I understand this should be fixed by your PR #872; let me know if I need to reopen.
